docs/general/blockchain-security/create-wallet.md

```diff
@@ -0,0 +1,75 @@
+---
+displayed_sidebar: generalSidebar
+---
+# Security Advice for Creating a Wallet
+
+The first step into the blockchain world is creating a secure wallet. This guide provides crucial security advice for creating your wallet.
+
+## 1. Understanding the Essence of Wallets
+
+A blockchain wallet is essentially a tool for managing your private keys. The private key is a randomly generated string of numbers, while the mnemonic phrase is a human-readable version of the private key. For example:
+
+- Private key: A seemingly random string of characters
+- Mnemonic phrase: Usually 12-24 easy-to-remember words
+
+Remember: Whoever holds the private key controls the assets in the wallet.
+
+## 2. Choosing the Right Wallet Type for Your Needs
+
+There are various wallet types, including:
```

Comment:
I suppose more introduction or relating articles should be added to this section. I would suppose the reader be a beginner of blockchain world and current version would make it hard for reader to understand

Reply:
Should add "what this is" as well as "what it is used for"